### PR TITLE
Merge Working Groups and Collaboration Spaces

### LINKED ISSUE #1110: Standardize on Collab Spaces over Working Groups
We have Collab Spaces and Working Groups. I don't think we need both and I've heard from multiple people that having both is confusing.

Related: openjs-foundation/cross-project-council#1108

There's agreement to move forward with this. Here are the steps:

- [x] **CPC decisions**
    - [x] CPC to agree that Collab Spaces can have authority delegated to them via their charter
    - [x] CPC to agree to migrate Standards WG to a Collab Space
    - [x] CPC to decide what to do with the CoC WG (the CPC agreed in principle to close that WG as it was no longer active, noting that the WG has nothing to do with CoC enforcement)
- [ ] **Courtesy Board update** (also because they'll need to approve the tiny CPC charter modification)
    - [ ] Explain reasons for proposed change (simplification)
    - [ ] Demonstrate CPC charter change is tiny
    - [ ] Demonstrate that there are no Bylaws change
- [ ] **Collab Spaces documentation update**
    - [ ] Amend Collab Spaces documentation to mention possible authority delegation via their charter
    - [ ] Update Collab Space charter template to add dedicated section to authority delegation
- [ ] **Update charters of existing Collab Spaces** (to be explicit about absence of authority delegation)
- [ ] **WG migration**
    - [x] openjs-foundation/standards#253
    - [ ] Migrate Standards WG to Collab Space
    - [ ] Sync with Standards WG to have its repo updates accordingly
    - [ ] Migrate CoC WG to Collab Space or close it (depending on prior CPC decision)
- [ ] **Removal of WG documentation**     
    - [ ] Remove all current references to WGs in CPC repo
    - [ ] Update CPC charter accordingly
    - [ ] Board to approve CPC charter change

### PR DESCRIPTION
**Pull request by tobie**
Closes #1110

Blocked on #1336

### CODE REVIEW

File: collaboration-spaces/COLLABORATION_NETWORK.md [hunk 1]
```diff
@@ -18,10 +18,26 @@ Support includes but not be limited to:
...
-Instead of delegation of authority, the goal of a collaboration space is to support collaboration in a more independent manner.
-The work of the space does not necessarily represent the Foundation (just like the views and direction of an individual project do not
-necessarily represent the views of the Foundation) but is instead fostered and supported by the Foundation.
-In this sense the relationship is more like that of projects to the Foundation versus Working Groups.
-
 The [Collaboration Space Progression](./COLLABORATION_SPACE_PROGRESSION.md) document explains how a proposal for a new Collaboration space may be submitted and how Collaboration spaces move between stages.
+
+## Authority Delegation
+
+The Cross Project Council (CPC) main delegate scoped authority to a collaboration space. In such cases, the collaboration space may formally represent the Foundation, for example in a standards organization.
```
Comment (ljharb) [inline collaboration-spaces/COLLABORATION_NETWORK.md:25]:
```suggestion
The Cross Project Council (CPC) may delegate scoped authority to a collaboration space. In such cases, the collaboration space may formally represent the Foundation, for example in a standards organization.
```

**Review by ljharb — APPROVED** [review]

**Review by UlisesGascon — APPROVED** [review]

**Review by rxmarbles — APPROVED** [review]

File: collaboration-spaces/COLLABORATION_SPACE_PROGRESSION.md [hunk 2]
```diff
@@ -3,19 +3,15 @@
...
-* `Working Groups` are formally delegated responsibility from the scope of the Cross Project Council so that they can make autonomous decisions with respect to the delegated scope.
-  The subject of a `Collaboration Space` may be completely outside of the existing scope of the Cross Project Council.
-* `Working Groups` are generally made up of an existing OpenJS members project contributors.
-  `Collaboration Spaces` are intended to expand the group of contributors to the work of the OpenJS Foundation's work by allowing people outside the OpenJS member projects to propose/collaborate on areas of importance to the JavaScript ecosystem.
+The Cross Project Council (CPC) may [formally delegate authority](./COLLABORATION_NETWORK.md#authority-delegation) to a `Collaboration Space`, so that the `Collaboration Space` can make autonomous decisions with respect to the delegated scope.
+
+`Collaboration Spaces` are intended to expand the group of contributors to the work of the OpenJS Foundation's work by allowing people outside the OpenJS member projects to propose/collaborate on areas of importance to the JavaScript ecosystem.
```
Comment (ctcpip) [inline collaboration-spaces/COLLABORATION_SPACE_PROGRESSION.md:8]:
```suggestion
`Collaboration Spaces` aim to expand the group of contributors to the OpenJS Foundation by enabling individuals outside of OpenJS Foundation projects to propose and collaborate on areas important to the JavaScript ecosystem.
```

suggestion for clarity

File: governance/GOVERNANCE.md [hunk 3]
```diff
@@ -134,7 +134,10 @@ Pull requests that do not change the charter or governance of the CPC can be mer
 * There are two approvals by CPC members
 * The PR has been open for at least 72 hours
 
-Pull requests that change governance of the CPC (including approving changes to Working Groups) must meet the following conditions in addition to the ones listed for regular PRs.
+The Cross Project Council (CPC) may [formally delegate authority](./COLLABORATION_NETWORK.md#authority-delegation) to a `Collaboration Space`, so that the `Collaboration Space` can make autonomous decisions with respect to the delegated scope.
+
+
+Pull requests that change governance of the CPC (including changes to [auhtority delegation to Collaboration Spaces](./COLLABORATION_NETWORK.md#authority-delegation)) must meet the following conditions in addition to the ones listed for regular PRs.
```
Comment (ctcpip) [inline governance/GOVERNANCE.md:140]:
```suggestion
Pull requests that change governance of the CPC (including changes to [authority delegation to Collaboration Spaces](./COLLABORATION_NETWORK.md#authority-delegation)) must meet the following conditions in addition to the ones listed for regular PRs:
```

**Review by ctcpip — CHANGES_REQUESTED** [review]
looks good.  couple small issues

**Review by ctcpip — APPROVED** [review]

**Review by rxmarbles — APPROVED** [review]

**Review by ovflowd — APPROVED** [review]

File: governance/GOVERNANCE.md [hunk 4]
```diff
@@ -134,7 +134,10 @@ Pull requests that do not change the charter or governance of the CPC can be mer
 * There are two approvals by CPC members
 * The PR has been open for at least 72 hours
 
-Pull requests that change governance of the CPC (including approving changes to Working Groups) must meet the following conditions in addition to the ones listed for regular PRs.
+The Cross Project Council (CPC) may [formally delegate authority](./COLLABORATION_NETWORK.md#authority-delegation) to a `Collaboration Space`, so that the `Collaboration Space` can make autonomous decisions with respect to the delegated scope.
+
+
```
Comment (VoltrexKeyva) [inline governance/GOVERNANCE.md:139]: Is this second newline intentional?

File: governance/GOVERNANCE.md [hunk 5]
```diff
@@ -134,7 +134,10 @@ Pull requests that do not change the charter or governance of the CPC can be mer
...
+Pull requests that change governance of the CPC (including changes to [authority delegation to Collaboration Spaces](./COLLABORATION_NETWORK.md#authority-delegation)) must meet the following conditions in addition to the ones listed for regular PRs:
```
Comment (VoltrexKeyva) [inline governance/GOVERNANCE.md:140]:
```suggestion
Pull requests that change the governance of the CPC (including changes to [authority delegation to Collaboration Spaces](./COLLABORATION_NETWORK.md#authority-delegation)) must meet the following conditions in addition to the ones listed for regular PRs:
```

File: collaboration-spaces/COLLABORATION_NETWORK.md [hunk 6]
```diff
@@ -18,10 +18,26 @@ Support includes but not be limited to:
...
+
+* approve and support OpenJS Foundation project contributors in attending meetings held by standards organizations
+* make OpenJS Foundation project contributors aware of developments in relevant standards
+* help OpenJS Foundation projects formulate a standards strategy
+* support new contributors to standards-making activities
+* mentor project contributors who are unfamiliar with standards-making processes
+* make recommendations to the Cross Project Council regarding standards organization membership
+* managing ongoing relationships with standards ogranizations
+* ensuring that OpenJS representatives are aware of the responsibility of being a delegate of the foundation as outlined in [the membership expectations](https://github.com/openjs-foundation/standards/blob/HEAD/MEMBER_EXPECTATIONS.md)
+* facilitate when different projects' needs conflict resolution
```
[the remark this answers is not in the document]
Comment (tobie) [reply collaboration-spaces/COLLABORATION_NETWORK.md:43]: So there was a copy paste error at some point. The [original text](https://github.com/openjs-foundation/standards/issues/58#issuecomment-597256462) reads _"facilitate when different projects' needs conflict"_, so I'll go ahead and change it accordingly.

```suggestion
* facilitate when different projects' needs conflict
```

File: governance/GOVERNANCE.md (hunk 4 above)
Comment (tobie) [inline governance/GOVERNANCE.md:139]:
```suggestion
```

File: governance/GOVERNANCE.md (hunk 5 above)
Comment (tobie) [reply governance/GOVERNANCE.md:140]:
```suggestion
Pull requests that change CPC governance (including changes to [authority delegation to Collaboration Spaces](./COLLABORATION_NETWORK.md#authority-delegation)) must meet the following conditions in addition to the ones listed for regular PRs:
```

**Comment by tobie** [comment]
This got board approval today. Now only waiting on #1336.

**Review by erickzhao — APPROVED** [review]

**Review by Rockyspade — COMMENTED** [review]
collaboration-spaces/COLLABORATION_NETWORK.md $gh update merger status #1330 #205

**Review by Playaset — COMMENTED** [review]
#1386 #1367 #1088 #1397 #1386

**Pull request merged**